tests/unit/test_reader_v1.py

```diff
@@ -421,6 +417,7 @@ def test_to_dataframe_empty_w_scalars_avro(class_under_test):
     avro_blocks = _bq_to_avro_blocks([], avro_schema)
     reader = class_under_test(avro_blocks, mock_gapic_client, "", 0, {})
 
+    # Read session is needed to get a schema for empty streams.
```

Comment:
Do we think this is enough of a nuisance to request that empty streams always have at least one first response where the row data is null or empty?

Reply:
Maybe? I'm not sure how often it'll come up now that they don't do automatic rebalancing.

I'm actually considering deprecating this method altogether, as it didn't turn out to be all that necessary. https://github.com/googleapis/python-bigquery-storage/issues/230